package.json

```diff
@@ -27,14 +27,18 @@
     "prepare-translation-status": "tsx scripts/prepare-translation-status.ts",
     "generate-pwa-icons": "tsx scripts/generate-pwa-icons.ts",
     "postinstall": "ignore-dependency-scripts \"stale-dep -u && simple-git-hooks && nuxi prepare && nr prepare-translation-status\"",
-    "release": "stale-dep && bumpp && tsx scripts/release.ts"
+    "release": "stale-dep && bumpp && tsx scripts/release.ts",
+    "build:glean": "glean translate telemetry/metrics.yaml telemetry/pings.yaml -f typescript -o telemetry/generated",
+    "docs:glean": "glean translate telemetry/metrics.yaml pings.yaml -f markdown -o telemetry/generated/docs",
+    "lint:glean": "glean glinter telemetry/metrics.yaml telemetry/pings.yaml"
   },
   "dependencies": {
     "@emoji-mart/data": "^1.1.2",
     "@fnando/sparkline": "^0.3.10",
     "@iconify-emoji/twemoji": "^1.0.2",
     "@iconify/json": "^2.2.85",
     "@iconify/utils": "^2.1.7",
+    "@mozilla/glean": "^1.4.0",
```

Comment:
A more recent version of glean is out, it's strongly recommended to start with that one :)